tests/phpunit/integration/Core/Tags/Guards/WP_Query_404_GuardTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * WP_Query_404_GuardTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Tags\Guards
+ * @copyright 2023 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Tags\Guards;
+
+use Google\Site_Kit\Core\Tags\Guards\WP_Query_404_Guard;
+use Google\Site_Kit\Tests\TestCase;
+
+class WP_Query_404_GuardTest extends TestCase {
+
+	/**
+	 * Tag_Guard object.
+	 *
+	 * @var WP_Query_404_Guard
+	 */
+	private $guard;
+
+	public function set_up() {
+		parent::set_up();
+		$this->guard = new WP_Query_404_Guard();
```

Comment:
This makes more sense to do when the class has more dependencies and/or otherwise repetitive setup. This can be simply inlined in the tests.